build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -25,11 +45,28 @@ data:
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
     $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+    $OVSDriverDir="$mountPath\openvswitch\driver"
+    $WorkDir="$mountPath\openvswitch\"
+
+    # Check if OVSExt driver is already installed
+    $driverStatus = netcfg -q ovsext                                                                           
+    if ($driverStatus -like '*not installed*') { 
+    # Install OVS Driver
+    net start vmms
+    cd $mountPath/openvswitch/driver
+    netcfg -l .\ovsext.inf -c s -i OVSExt
```

Comment:
According to your test, is this step (netcfg -i OVSExt ) always required after we restart Pod/host or it happens only once ? @NamanAg30

Reply:
netcfg -i OVSExt happens when driver is not installed .... so whenever the pod restarts it will first check the driver status and then this step will be executed accordingly.

Reply:
I know your patch is try to install the driver as long as it is not installed, I am asking about the observations in your test that is the previously installed driver still working correctly after Pod restart? And does it work after host reboot?

Reply:
I have verified both cases i.e od restart and host reboot . In both cases , the previously installed driver is working.